x-pack/plugin/core/src/test/java/org/elasticsearch/license/AbstractLicensesIntegrationTestCase.java

```diff
@@ -26,7 +26,17 @@
 
     @Override
     protected Settings nodeSettings(int nodeOrdinal) {
-        return Settings.builder().put(super.nodeSettings(nodeOrdinal)).put(XPackSettings.SECURITY_ENABLED.getKey(), false).build();
+        return customSettings(super.nodeSettings(nodeOrdinal));
+    }
+
+    @Override
+    protected Settings transportClientSettings() {
+        // Plugin should be loaded on the transport client as well
```

Comment:
Is this comment out of place? I understand it is carried over from the previous method. But should it be in the `transportClientPlugins` method?